lib/ziti-tunnel-cbs/include/ziti/ziti_tunnel_cbs.h

```diff
@@ -307,6 +308,14 @@ struct add_identity_request_s {
     void *cmd_ctx;
 };
 
+#ifndef IPC_SUCCESS
```

Comment:
why protect these in `#ifdef`s? do we want to allow user-supplied values of IPC_SUCCESS / IPC_ERROR?

Reply:
these are constant fields. I will remove the ifdef